src/util/scroll-spy.js

```diff
@@ -334,7 +334,7 @@
 
 				// Get the content for the nav item
                 const target = item.getAttribute(options.attribute)
-				const id = decodeURIComponent(target.substr(1));
+				const id = decodeURIComponent(target?.substr(1));
```

Comment:
Happy with the change. But would like to know why the target is null?

Reply:
The element does not have a "data-slug" attribute on it. (Target, and ID are both the "data-slug" attribute on the elements)
<img width="876" alt="Screenshot 2023-09-27 at 08 07 10" src="https://github.com/SPANDigital/presidium-js/assets/34862470/eb352de0-8dfa-4bfa-be4a-ed1d817768be">

---

From my experience working with Presidium, I haven't come across a file being referenced this way, and I'm not entirely sure why it does not have a "data-slug". This seems like an issue unique to this repo, thoughts?
<img width="471" alt="Screenshot 2023-09-27 at 08 07 59" src="https://github.com/SPANDigital/presidium-js/assets/34862470/6592c912-0283-4d8c-82fe-0a26bd13b348">